package.json

```diff
@@ -25,47 +25,47 @@
     "@google-cloud/profiler": "^4.2.0",
     "@google-cloud/trace-agent": "^5.1.6",
     "@rollup/plugin-babel": "^6.0.3",
-    "@rollup/plugin-node-resolve": "^15.1.0",
+    "@rollup/plugin-node-resolve": "^15.2.1",
     "@rollup/plugin-replace": "^5.0.2",
     "@rollup/plugin-terser": "^0.4.3",
     "@swc-node/register": "^1.6.6",
     "@swc/core": "^1.3.70",
     "@types/express": "^4.17.17",
     "@types/express-winston": "^4.0.0",
-    "@types/jest": "^29.5.3",
+    "@types/jest": "^29.5.4",
     "@types/jest-when": "^3.5.2",
     "@types/jsdom": "^21.1.1",
     "@types/superagent": "^4.1.16",
     "@types/winston": "^2.4.4",
-    "@typescript-eslint/eslint-plugin": "^5.62.0",
-    "@typescript-eslint/parser": "^5.62.0",
-    "babel-jest": "29.6.4",
+    "@typescript-eslint/eslint-plugin": "^6.5.0",
+    "@typescript-eslint/parser": "^6.5.0",
+    "babel-jest": "^29.6.4",
     "babel-watch": "^7.8.1",
     "eslint": "^8.48.0",
-    "eslint-config-prettier": "^8.8.0",
-    "eslint-import-resolver-typescript": "^3.5.5",
+    "eslint-config-prettier": "^9.0.0",
```

Comment:
FYI I ran into issues updating this in webapp.  I think it conflicted with dprint or something like that.

Reply:
We don't use dprint here and it's only a dev dependency so there shouldn't be any issues making the upgrade in this repo.